res/fingerprint-server-api.yaml

```diff
@@ -323,8 +324,7 @@ paths:
           description: >-
             Unique [visitor
             identifier](https://dev.fingerprint.com/reference/get-function#visitorid)
-            issued by Fingerprint Pro.
-          x-go-skip-pointer: true
+            issued by Fingerprint Pro.\
```

Comment:
why is there `\` in the end?

Reply:
Schema typo. Fix for OpenAPI schema is [here](https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/pull/199), I've also pushed commit with fix to the SDK, so we won't have to release it separately.